syntax = "proto3";

package auth;

service AuthService {
  rpc ValidateUser (ValidateUserRequest) returns (ValidateUserResponse);
}

message ValidateUserRequest {
  string email = 1;
}

message ValidateUserResponse {
  bool is_valid = 1;
  string username = 2;
}
